dbclient/jdbc/src/main/java/io/helidon/dbclient/jdbc/JdbcStatementQuery.java

```diff
@@ -66,7 +67,7 @@
 
         executeContext().addFuture(queryFuture);
 
-        return dbContextFuture.thenCompose(interceptorContext -> {
+        CompletionStage<DbRows<DbRow>> result = dbContextFuture.thenCompose(interceptorContext -> {
```

Comment:
If you rebase on latest master, you can use Java 11 and `var`, which may be nicer when we use these complex generics (not required)

Reply:
Fixed.

Reply:
I would like to keep full types here now. It's questionable whether code will be better readable with var or full type information.